src/SFA.DAS.Commitments.Api.Types/SFA.DAS.Commitments.Api.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
     <metadata>
         <id>SFA.DAS.Commitments.Api.Types</id>
-        <version>1.71.0</version>
```

Comment:
As above the api.types project is referenced directly in the solution, so the nuget package should not be used